### PR TITLE
61 holler image component

### LINKED ISSUE #61: Build holler/image component
Wireframe: https://makersacademy.mybalsamiq.com/projects/newsitewireframes/holler%20block%20with%20image

Required for #9.

### PR DESCRIPTION
**Pull request by sjmog**
Closes #61, needed for the Curriculum page

### CODE REVIEW

**Comment by sjmog** [comment]
@pitchinvasion OI OI OI OI OI MERGE IT MERGE IT MERGE IT OI OI OI

File: source/components/groups/_title-block.erb [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
   <% end %>
 
-  <% if defined?(body) %>
+  <% if defined?(body) && !body.nil? %>
```
Comment (pitchinvasion) [inline source/components/groups/_title-block.erb]: Was there a specific use case with body being defined but specifically defined as nil? Just checking this isn't unneccesarily defensive programming

File: source/components/templates/_slider.erb [hunk 2]
```diff
@@ -1,7 +1,3 @@
 <% component = "slider" %>
 
-<% if type == "media-components" %>
-
-<%= partial "templates/slider/media-components", locals: locals %>
-
-<% end %>
\ No newline at end of file
+<%= partial "templates/slider/#{type}", locals: locals %>
```
Comment (pitchinvasion) [inline source/components/templates/_slider.erb]: image

**Pull request merged**